protocol/v2/ssv/validator/msgqueue_consumer.go

```diff
@@ -22,6 +22,8 @@ type MessageHandler func(logger *zap.Logger, msg *queue.SSVMessage) error
 // queueContainer wraps a queue with its corresponding state
 type queueContainer struct {
 	Q          queue.Queue
+	StopQueueF context.CancelFunc
```

Comment:
i'd recommend renaming it to just Stop or Cancel for a bit more elegance

Reply:
Agree. Thanks